cdap-app-fabric/src/test/java/io/cdap/cdap/internal/app/services/http/handlers/AppScheduleUpdateTest.java

```diff
@@ -109,7 +109,7 @@ public void testUpdateSchedulesFlag() throws Exception {
     actualSchSpecs = listSchedules(TEST_NAMESPACE_META2.getNamespaceId().getNamespace(),
                                    defaultAppId.getApplication());
 
-    // workflow is deleted, so the schedules will be deleted now
-    Assert.assertEquals(0, actualSchSpecs.size());
+    // workflow is deleted, however the schedules will not be deleted since they are carried over from older version
```

Comment:
so if the workflow is removed, does it mean the existing schedules are deleted? If thats the case then it's a bug and needs to be addressed

Reply:
it looks like that's what this is testing. It makes sense, as if we kept the schedule, it would be a schedule that always fails since there is no corresponding workflow to run.

Reply:
Got it. Addressing this in https://cdap.atlassian.net/browse/CDAP-20403 before I merge this current PR